src/components/v2v3/shared/FundingCycleConfigurationDrawers/DistributionSplitModal/V2ProjectPayoutFormItem.tsx

```diff
@@ -0,0 +1,62 @@
+import { t, Trans } from '@lingui/macro'
+import * as constants from '@ethersproject/constants'
+import { Form, InputNumber, Radio } from 'antd'
+import { stringIsDigit } from 'utils/math'
+import { featureFlagEnabled } from 'utils/featureFlags'
+import { FEATURE_FLAGS } from 'constants/featureFlags'
+
+export const V2_ALLOCATOR_ADDRESS = 'TODO_V2_ALLOCATOR_ADDRESS'
+const V3_ALLOCATOR_ADDRESS = constants.AddressZero
+
+export function V2ProjectPayoutFormItem({
+  value,
+  onChange,
+}: {
+  value: string | undefined
+  onChange: (projectId: string | undefined) => void
+}) {
+  const validateProjectId = () => {
+    if (!stringIsDigit(value ?? '')) {
+      return Promise.reject(t`Project ID must be a number.`)
+    }
+    // TODO: check if projectId exists
+    return Promise.resolve()
+  }
+
+  const allocatorsEnabled = featureFlagEnabled(FEATURE_FLAGS.SPLIT_ALLOCATORS)
+
+  return (
+    <>
+      {allocatorsEnabled ? (
```

Comment:
**issue** this should only be shown if CV == 2.

Reply:
Got it